doc/conf.py

```diff
@@ -42,6 +42,11 @@
     print "ipython: %s, %s" % (IPython.__version__, IPython.__file__)
 except ImportError:
     print "no ipython"
+try:
+    import seaborn
```

Comment:
That is really odd. However, I just built the docs and was not able to reproduce your described behavior.

What happens if you put `seaborn.reset_orig()`  after the import.